privacyidea/lib/tokens/emailtoken.py

```diff
@@ -417,7 +418,7 @@ def _get_auto_email(options):
         return autosms
 
     @log_with(log)
-    def _compose_email(self, message="<otp>", subject="Your OTP", mimetype="plain"):
+    def _compose_email(self, options, message="<otp>", subject="Your OTP", mimetype="plain"):
```

Comment:
I recommend to add new parameters in a function always at the end as keyword parameters.
This way you can avoid to run into problems if there are function calls, that you do not update.

~~~~python
def _compose_email(self, message="<otp>", subject="Your OTP", mimetype="plain", options=None):
~~~~

Also, do not use empty dicts as default values.
This is why you need to initialize the options, in case it is not set. (see below)